src/main/kotlin/org/opensearch/commons/alerting/model/Alert.kt

```diff
@@ -43,6 +43,7 @@ data class Alert(
     val aggregationResultBucket: AggregationResultBucket? = null,
     val executionId: String? = null,
     val associatedAlertIds: List<String>,
+    val clusters: List<String>? = null,
```

Comment:
will this break security analytics constructor?? if yes, plz create PR for that too

Reply:
what is stored here? can you share an example list of opensource clusters?

Reply:
should this be called clusterNames or clusterIds

Reply:
Created this PR to update SAP https://github.com/opensearch-project/security-analytics/pull/837